cli/command/trust/info.go

```diff
@@ -101,9 +101,50 @@ func lookupTrustInfo(cli command.Cli, remote string) error {
 	}
 	printSignatures(cli, remote, signatureRows)
 
+	roleWithSigs, err := notaryRepo.ListRoles()
+	if err != nil {
+		fmt.Fprintf(cli.Out(), "Error listing signature metadata for roles: %s\n", err)
+		return err
+	}
+
+	signerRoleToKeyIDs, baseRoleToKeyIDs := getSignerAndBaseRolesWithKeyIDs(roleWithSigs)
+
+	fmt.Fprintf(cli.Out(), "\nList of signers and their KeyIDs:\n\n")
+	printRolesWithKeyIDs(cli, signerRoleToKeyIDs)
+
+	fmt.Fprintf(cli.Out(), "\nList of admins and their KeyIDs:\n\n")
+	printRolesWithKeyIDs(cli, baseRoleToKeyIDs)
+
 	return nil
 }
 
+// Extract signer keys and base keys from the list of roles
```

Comment:
nit: could we rename `base` to `admin` or another name here?  I think it's important to note that the only keys we display in that map are offline admin keys (ex: no `timestamp`)